docs/source/getting_started/index.rst

```diff
@@ -44,29 +44,17 @@ Starting PostgreSQL Container
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Now we need to run ``PostgreSQL`` in a container, attach it to the network you
-have created before, and expose ports for communication:
+have created before:
 
 .. code-block:: shell
 
   docker run --name some-postgres \
   -e POSTGRES_USER=postgres \
   -e POSTGRES_PASSWORD=mysecretpassword \
-  -p 5432:5432 \
```

Comment:
Will it work w/o exposing the port? Also, what if user has something on this port?

Reply:
Since both Iroha and Postgres containers in the same `iroha-network`, it will work without exposing ports on the host. I also put a note about exposing Postgres port to 127.0.0.1 for debugging purposes. In this case it should not be used by any other program